apps/server/src/app/__tests__/utils/user.ts

```diff
@@ -3,12 +3,12 @@ import prisma from '../../lib/prisma'
 
 const EMAIL = 'test@example.com'
 
-export async function resetUser(auth0Id = '__TEST_USER_ID__'): Promise<User> {
+export async function resetUser(authId = '__TEST_USER_ID__'): Promise<User> {
     const [_, [user]] = await prisma.$transaction([
-        prisma.$executeRaw`DELETE FROM "user" WHERE auth0_id=${auth0Id}`,
```

Comment:
Not sure changing in this way is correct? We likely want to rely on userId instead of any "authId" moving forward?

Reply:
I believe @tmyracle can speak to if this is correct or not.

Reply:
Yes this is correct. the AuthUser object handles the base information used by NextAuth and is associated with the pre-existing User object via with auth_id field, similar to how it was handled with Auth0.